Alignment/APEEstimation/test/cfgTemplate/apeEstimator_cfg.py

```diff
@@ -0,0 +1,331 @@
+process.MessageLogger.categories.append('AlignmentTrackSelector')
+process.MessageLogger.cerr.INFO.limit = 0
+process.MessageLogger.cerr.default.limit = -1
+process.MessageLogger.cerr.SectorBuilder = cms.untracked.PSet(limit = cms.untracked.int32(-1))
+process.MessageLogger.cerr.HitSelector = cms.untracked.PSet(limit = cms.untracked.int32(-1))
+process.MessageLogger.cerr.CalculateAPE = cms.untracked.PSet(limit = cms.untracked.int32(-1))
+process.MessageLogger.cerr.ApeEstimator = cms.untracked.PSet(limit = cms.untracked.int32(-1))
+process.MessageLogger.cerr.AlignmentTrackSelector = cms.untracked.PSet(limit = cms.untracked.int32(-1))
+process.MessageLogger.cerr.FwkReport.reportEvery = 1000 ## really show only every 1000th
+
+
+
+##
+## Process options
+##
+process.options = cms.untracked.PSet(
+    wantSummary = cms.untracked.bool(True),
+)
+
+
+
+##
+## Input sample definition
+##
+isData1 = isData2 = False
+isData = False
+isQcd = isWlnu = isZmumu = isZtautau = isZmumu10 = isZmumu20 = isZmumu50 = False
+isMc = False
+isParticleGunMuon = isParticleGunPion = False
+isParticleGun = False
+if options.sample == 'data1':
+    isData = True
+elif options.sample == 'data2':
+    isData = True
+elif options.sample == 'data3':
+    isData = True
+elif options.sample == 'data4':
+    isData = True
+elif options.sample == 'qcd':
+    isMc = True
+elif options.sample == 'wlnu':
+    isMc = True
+elif options.sample == 'zmumu':
+    isMc = True
+elif options.sample == 'ztautau':
+    isMc = True
+elif options.sample == 'zmumu10':
+    isMc = True
+elif options.sample == 'zmumu20':
+    isMc = True
+elif options.sample == 'zmumu50':
+    isMc = True
+else:
+    print 'ERROR --- incorrect data sammple: ', options.sample
+    exit(8888)
+
+
+##
+## Input Files
+##
+readFiles = cms.untracked.vstring()
+process.source = cms.Source ("PoolSource",
+    fileNames = readFiles
+)
+readFiles.extend( [
+    'file:reco.root',
+] )
+
+
+
+##
+## Number of Events (should be after input file)
+##
+process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(-1) )
+
+
+
+##
+## Check run and event numbers for Dublicates --- only for real data
+##
+process.source.duplicateCheckMode = cms.untracked.string("checkEachRealDataFile")
+#process.source.duplicateCheckMode = cms.untracked.string("checkAllFilesOpened")   # default value
+
+
+
+##
+## Whole Refitter Sequence
+##
+process.load("Alignment.APEEstimation.TrackRefitter_38T_cff")
+
+if isParticleGun:
+    process.GlobalTag.globaltag = 'DESIGN42_V12::All'
+elif isMc:
+    process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_design', '')
+    
+    ##### To be used when running on Phys14MC with a CMSSW version > 72X
+    process.GlobalTag.toGet = cms.VPSet(
+		cms.PSet(
+			record = cms.string("BeamSpotObjectsRcd"),
+			tag = cms.string("Realistic8TeVCollisions_START50_V13_v1_mc"),
+			connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS')
+		)
+	)
+
```

Comment:
@cschomak this customization is needed only for the 72X Phys14 datasets, please comment it.